nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -261,6 +261,8 @@
 
 - Setting `nixpkgs.config` options while providing an external `pkgs` instance will now raise an error instead of silently ignoring the options. NixOS modules no longer set `nixpkgs.config` to accomodate this. This specifically affects `services.locate`, `services.xserver.displayManager.lightdm.greeters.tiny` and `programs.firefox` NixOS modules. No manual intervention should be required in most cases, however, configurations relying on those modules affecting packages outside the system environment should switch to explicit overlays.
 
+- The `services.locate.locate` has been renamed to `services.locate.package` to align with the convention of other services.
+
```

Comment:
I don't think we need this. There are no breaking changes and the warning with the renaming should be enough.

Reply:
Thanks for the quick review! I've removed the release note :+1: